projects/common/src/time/time-duration.ts

```diff
@@ -111,6 +178,10 @@ export class TimeDuration {
     }
   }
 
+  private buildMilitaryTime(): Date {
+    return new Date(this.millis);
+  }
```

Comment:
what is military time? why is it required in a duration object?

this.millis is a number capturing the duration. it has nothing to do with a specific time. here, its used to create a timestamp. thats the equivalent of saying 3 hours can be converted into a specific timestamp, which is not possible.